pyproject.toml

```diff
@@ -247,6 +282,16 @@ extend-ignore = [
     # flake8-print (T20)
     "T201",  # PrintUsed                             # TODO: change to logging
 
+    # tryceratops (TRY)
+    "TRY002",  # raise-vanilla-class                 # NOTE: low priority fix, requiring API change
+    "TRY003",  # raise-vanilla-args
+    "TRY004",  # prefer-type-error                   # TODO: fix, but breaks API
+    "TRY200",  # TRY200                              # TODO: fix
```

Comment:
?? What is `TRY200`?

Reply:
Added the description (reraise-no-cause)